frontend/awx/resources/inventories/hooks/useDeleteInventories.tsx

```diff
@@ -29,6 +37,18 @@ export function useDeleteInventories(onComplete: (inventories: Inventory[]) => v
       actionColumns,
```

Comment:
The count of inventories in the confirmation message `Yes, I confirm that I want to delete these {count} inventories` does not reflect non-deletable inventories. The count on [this line](https://github.com/ansible/ansible-ui/pull/379/files#diff-7cd8e950c6f599e92b82548e296642a9e0727b5727047c3283e8fc8cdf5ac1c3R30) should be `inventories.length - undeletableInventories.length`